backend/app.py

```diff
@@ -432,19 +442,20 @@ def data_stuff_for_codeset_ids(codeset_ids):
                 selected= lambda row: row.codeset_id.isin(codeset_ids))
         .sort_values(by=['selected', 'concepts'], ascending=False)
     )
-
-    researcher_cols = ['created_by_container', 'created_by_version',
-                       'assigned_sme', 'reviewed_by', 'n3c_reviewer',
-                       'assigned_informatician', ]
-
     dsi.selected_csets = dsi.related_csets[dsi.related_csets['codeset_id'].isin(codeset_ids)]
 
-    # works but is currently too slow. commenting out till ready to make faster
-    # researchers = []
-    # for (i, row) in dsi.selected_csets.iterrows():
-    #   researchers.append({col: get_researcher(row[col]) for col in researcher_cols if row[col]})
-    # dsi.selected_csets['researchers'] = researchers
-
+    # Researchers
+    researcher_cols = ['created_by_container', 'created_by_version', 'assigned_sme', 'reviewed_by', 'n3c_reviewer',
```

Comment:
These are the changes in `data_stuff_for_codeset_ids()` related to researchers. Now it is getting data from the downloaded CSV rather than making a request.

Reply:
Here's a snippet of what the results look like:

Example: http://127.0.0.1:8000/cr-hierarchy?rec_format=flat&codeset_id=400614256|411456218|419757429|484619125|818292046|1036505

```json
{
	"selected_csets": [
	{
		"codeset_id": 1036505,
		"concept_set_version_title": "N3C Paroxetine (v1)",
		...
		"selected": true,
		"researchers": {
			"multipassId": "21131bdf-956a-4ff6-973e-96ba46a2d7f1",
			"institutionsId": "",
			"name": "Anna OMalley",
			"emailAddress": "annao@palantir.com",
			"unaPath": "",
			"signedDua": "",
			"citizenScientist": "",
			"internationalScientistWithDua": "",
			"institution": "",
			"orcidId": "",
			"rid": "ri.phonograph2-objects.main.object.8205dae4-e2d6-443b-a0c4-f345a4f38e8e"
		},
	},
	...	
}
```

Reply:
The CSV is also now committed into `termhub-csets`

Reply:
Nice!